src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -179,11 +179,13 @@ class FacetSidebar extends React.Component {
                 return (
                   <label
                     key={j}
+                    id={`${facet.field}-${valueLabel}`}
                     htmlFor={`${facet.field}-${valueLabel}`}
                     className={`nypl-bar_${percentage}`}
                   >
                     <input
                       id={`${facet.field}-${valueLabel}`}
+                      aria-labelledby={`${facet.field} ${valueLabel}`}
```

Comment:
yes... i understand... in the example i give you the text “fiction” is inside a `span`